staff/pep-delbano/js-arrays/closures/salute.js

```diff
@@ -0,0 +1,29 @@
+function salute(who) {
```

Comment:
I'm pretty sure this file should not be here

Reply:
= true!
I delete that folder right now!